tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -526,22 +481,33 @@ def update_schema(self, name: str = None, description: str = None):
         :returns: a success or error message
         :rtype: string
         """
+        if name is None:
```

Comment:
I see this check in a few places. Perhaps move this into a new method?

Reply:
I simplified this check, reducing the number of lines required from 9 to 3 for each check in the latest commit